### ISSUE TITLE
Can't install Horovod with newest TF because of error: ‘bfloat16’ in namespace ‘Eigen’ does not name a type

### ISSUE THREAD

**Issue by Zantares** [horovod/horovod#2232]
**Environment:**
1. Framework: TensorFlow
2. Framework version: commit id(64fa2fe6ce8582582210806acf3d78d423cde196)
3. Horovod version: 0.19.5
4. MPI version: 3.1
5. CUDA version: N/A
6. NCCL version: N/A
7. Python version: 3.6
8. Spark / PySpark version: N/A
9. OS and version: 
10. GCC version: 7.4.0

**Checklist:**
1. Did you search issues to find if somebody asked this question before?
2. If your question is about hang, did you read [this doc](https://github.com/horovod/horovod/blob/master/docs/running.rst)?
3. If your question is about docker, did you read [this doc](https://github.com/horovod/horovod/blob/master/docs/docker.rst)?
4. Did you check if you question is answered in the [troubleshooting guide](https://github.com/horovod/horovod/blob/master/docs/troubleshooting.rst)?

**Bug report:**
try to pip install Horovod with source build TensorFlow, but an error was reported:
```
error: ‘bfloat16’ in namespace ‘Eigen’ does not name a type
   typedef Eigen::bfloat16 bfloat16;
```
I believe it's caused by missing build dependency for Horovod after TF upgrade the usage of Eigen::bfloat16. Maybe need to fix BUILD file somewhere in Horovod.

**Comment by tgaddair** [comment]
Hey @Zantares, this was resolved in #2184.  To workaround, you can install from master, or wait for the upcoming v0.20.0 release, which should hopefully come out later today.

**Comment by Zantares** [comment]
> Hey @Zantares, this was resolved in #2184. To workaround, you can install from master, or wait for the upcoming v0.20.0 release, which should hopefully come out later today.

Thanks for the answer, I will use master first.

**Comment by tgaddair** [comment]
@Zantares, we actually just released v0.20.0 today, so feel free to use that.
